Apps/Medication/src/Migrations/20191016163758_InitialCreate.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Medication.Migrations
```

Comment:
HealthGateway.Medication.Database.Migrations

Reply:
This is the default from the tooling and uses the folder structure.  We could change this but would have to customize the tooling command each time we run.